### PR TITLE
Add ansible-lint workflow

### PR DESCRIPTION
**Pull request by ben-alkov**
I'm concerned about the 'role_name' & 'namespace' fields in 'meta/main.yml' - I don't know what's *supposed* to go there; I simply guessed.

Tested in ben-alkov/ansible-role-osbs-namespace PR 1 [linter run](https://github.com/ben-alkov/ansible-role-osbs-namespace/runs/4070539455?check_suite_focus=true)

Signed-off-by: Ben Alkov <ben.alkov@redhat.com>

- CLOUDBLD-7849

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
   license: BSD
   min_ansible_version: 2.1
   platforms:
-    name: EL
-    versions:
-    - 7
+    -
+      name: EL
+      versions:
+        - 7
```
Comment (chmeliik) [inline meta/main.yml:14]: Would this pass yamllint?
```suggestion
    - name: EL
      versions:
        - 7
```

**Comment by chmeliik** [comment]
> I'm concerned about the 'role_name' & 'namespace' fields in 'meta/main.yml' - I don't know what's _supposed_ to go there; I simply guessed.

The role name looks correct based on https://galaxy.ansible.com/docs/contributing/creating_role.html, I didn't find much about 'namespace'. Seems reasonable though

File: meta/main.yml (hunk 1 above)
Comment (ben-alkov) [reply meta/main.yml]: Ah, it was saying "meta-no-info: Platforms should be a list of dictionaries", but I didn't **completely** understand what it wanted. Your suggestion is good.

**Comment by ben-alkov** [comment]
Latest [GH action run](https://github.com/ben-alkov/ansible-role-osbs-namespace/runs/4070658080?check_suite_focus=true) from my fork

**Review by MartinBasti — APPROVED** [review]

**Review by mkosiarc — APPROVED** [review]

**Pull request merged**